tests/components/spotify/snapshots/test_media_player.ambr

```diff
@@ -45,9 +45,9 @@
       'media_artist': 'Rush',
       'media_content_id': 'spotify:track:4e9hUiLsN4mx61ARosFi7p',
       'media_content_type': <MediaType.MUSIC: 'music'>,
-      'media_duration': 296.466,
+      'media_duration': 296,
```

Comment:
Does the new library round timestamps? Why does it do that?

Reply:
The signature of media_duration is `int | None`, and we now round to adhere to the typing

Reply:
Apparently it was only my IDE complaining, not mypy